gzipi/cli.py

```diff
@@ -195,7 +199,7 @@ def _exists(path):
 
 
 def _repack_subparser(subparsers):
-    desc = 'Repack a gzipped file into a chunked gzipped file and an index file.'
+    desc = 'Repack a compressed gzip or zstd file into a chunked gzipped file and an index file.'
```

Comment:
This is inconsistent. gzip/zstd on LHS, gzip only on RHS.

```suggestion
    desc = 'Repack a compressed file into chunks supported by an index file.'
```